source/locale/pt_BR/LC_MESSAGES/releases.po

```diff
@@ -85,107 +84,110 @@ msgstr ""
 
 #: ../../source/releases.rst:28
 msgid "New node Event Device In"
-msgstr ""
+msgstr "Novo Nó Evento Dispositivo (Entrada)"
 
 #: ../../source/releases.rst:29
+#, fuzzy
 msgid "New node Event Template Device"
-msgstr ""
+msgstr "Novo nó Evento de modelos de dispositivo (Entrada)"
 
 #: ../../source/releases.rst:30
 msgid "New node FTP"
-msgstr ""
+msgstr "Novo nó FTP"
 
 #: ../../source/releases.rst:31
 msgid "New node Multi Device Out"
-msgstr ""
+msgstr "Novo nó de Saída para  vários dispositivos (Multi Device Out)"
 
 #: ../../source/releases.rst:32
-#, fuzzy
 msgid "New node Notification"
-msgstr "Notificações"
+msgstr "Novo nó de Notificações"
 
 #: ../../source/releases.rst:33
 msgid "New node Multi Actuate"
-msgstr ""
+msgstr "Novo nó de multiplias atuações (Multi Actuate)"
```

Comment:
"Novo nó de atuação para vários dispositivos (Multi Actuate)"
just to keep the standard